resources/views/layouts/app.blade.php

```diff
@@ -15,10 +15,10 @@
 
     @stack('head')
 </head>
-<body class="bg-white antialiased">
-    <div id="app">
+<body class="text-primary antialiased">
```

Comment:
This is a bit weird to have as default I think?

Reply:
I think its a good way of defining the default text color, this way you dont have to explicitly put on the primary text color on elements. The antialised is a thing we always use to make text look cleaner/sharper

Reply:
Primary will most likely be a striking color; let's say red. So using the primary color as text color isn't a good default.